core/lib/spree/testing_support/dummy_app/database.yml

```diff
@@ -1,33 +1,39 @@
-<%
-database_prefix = ENV['LIB_NAME'].presence || "solidus"
-%>
-<% case ENV['DB']
+<% db = ENV['DB'] %>
+<% db_prefix = ENV['LIB_NAME'].presence || "solidus" %>
+<% db_host = ENV['DB_HOST'] || ENV['DB'] %>
```

Comment:
Why this should be `ENV['DB']` if `ENV['DB_HOST']` is empty?

Reply:
Good question, I had forgotten to mention it in the PR description. This is kind of a trick to make it possible for the docker service to run both MySQL & Postgress suites through the usual `DB` environment variable. If we only use `DB_HOST` then we should force the `app` service to link either to `mysql` or `postgres` services or otherwise we would have to provide both `DB_HOST` & `DB` when running tests. Defaulting to `DB` when `DB_HOST` is not present, makes the docker container match the name of the database service name (which is the same as the name seen for it as a host from app).

Reply:
Thinking about it a second time, we can communicate better if we allow overriding `DB_HOST` through new `DB_MYSQL_HOST` & `DB_POSTGRES_HOST` env vars, respectively for MySQL and Postgres. This way we can define both on docker-compose.yml and we don't need to overuse `DB`. If you don't think otherwise I'll change it in this way.

Reply:
The result is not yet super clear to me but if you think it's better let's try, I'll take a look later after the change is made, thanks!

Reply:
Hey @kennyadsl , you can take a look at this commit:

https://github.com/solidusio/solidus/pull/3947/commits/cdf038ddfd3dcd0be2001461a383bfcfd9e27337

Reply:
Got it, I think it makes sense, even if that variables' definition is becoming too complex in my opinion. Maybe we can abstract that in some external class. Not saying we should do that in this PR, but maybe it's something we can start thinking about, wdyt?

Reply:
Yes, I completely agree.